src/D2L.CodeStyle.TestAnalyzers/ExpectedException/ExpectedExceptionAnalyzer.cs

```diff
@@ -51,6 +51,9 @@ public sealed class ExpectedExceptionAnalyzer : DiagnosticAnalyzer {
 					isExpectedException = true;
 					break;
 				} else if( attribute.Name.ToString().Equals( "TestCase" ) ) {
+					if( attribute.ArgumentList == null ) {

```

Comment:
When I built locally, I got the error message for LP. after I fix like this, the errors gone, I didn't track down to the specific code as the error message didn't show that...
>  [exec] CSC(0,0) : error AD0001: Analyzer 'D2L.CodeStyle.TestAnalyzers.TestCase.TestCaseAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'.
> [exec] CSC(0,0) : error AD0001: Analyzer 'D2L.CodeStyle.TestAnalyzers.ExpectedException.ExpectedExceptionAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'.